Source/cryptography/tests/test_key_generator/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+# If not stated otherwise in this file or this component's LICENSE file the
+# following copyright and licenses apply:
+#
+# Copyright 2023 Metrological
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+find_package(${NAMESPACE}Core REQUIRED)
+find_package(OpenSSL)
+
+add_executable(cgkeygen
+        Module.cpp
+        main.cpp)
+
+include_directories(${CMAKE_CURRENT_LIST_DIR}/../../../cryptography)
+include_directories($<INSTALL_INTERFACE:include/${NAMESPACE}>)
+
+set_target_properties(cgkeygen PROPERTIES
+        CXX_STANDARD 11
+        CXX_STANDARD_REQUIRED YES)
+
+target_link_libraries(cgkeygen
+        PRIVATE
+        ${NAMESPACE}Cryptography
```

Comment:
I always like consistency :-) and my brain got an error by this construction :-D
Why do we always have  ${NAMESPACE}Core::${NAMESPACE}Core and here only once (${NAMESPACE}Cryptography) without the semicolon. I understood from @bramoosterhuis that the first was the preferred way. 
If so lets stick to that and always use for components that we as metrological deliver the ${NAMESPACE}Cryptography::${NAMESPACE}Cryptography to avoid my brain errors :-)

Reply:
Fixed. Now need to educate myself on the difference. :)